packages/calcite-components/src/components/action-group/action-group.stories.ts

```diff
@@ -94,3 +94,26 @@ export const TurkishLocale_TestOnly = (): string =>
       <calcite-dropdown slot="menu-actions"> </calcite-dropdown>
     </calcite-action-group>
   </div>`;
+html`<div class="container">
```

Comment:
Is this dangling HTML snippet meant for a separate story? It is not currently being used.

Reply:
Not used. Removed here https://github.com/Esri/calcite-design-system/pull/10167